src/test/java/TestClass.java

```diff
@@ -0,0 +1,17 @@
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class TestClass {
+    @Test
+    public void generalTestCase() {
+        assertEquals(true, true);
```

Comment:
What is this testing? Perhaps a bit more documentation if you are actually expecting something here.